### ISSUE TITLE
OData query parameters support

### ISSUE THREAD

**Issue by antlio** [scalar/scalar#3926]
### What happens?

Request with OData query parameters return a 404:
![image](https://github.com/user-attachments/assets/38f233bf-cb5c-4e95-aa45-df91d04bdea4)

### What did you expect to happen?

_Originally posted by stan in [discord](https://discord.com/channels/1135330207960678410/1307587855559819386)

**Comment by antlio** [comment]
hey @standleypg, do you mind sharing a part of your document or do it within [our sandbox](https://sandbox.scalar.com/)? I don't reproduct

**Comment by antlio** [comment]
@standleypg hey sorry for the time taken to get back to you on this, do you still encounter the issue?

**Comment by standleypg** [comment]
Hey @antlio, terribly sorry for the late response. It seems the URL can't be editted anymore now.

**Comment by antlio** [comment]
> Hey [@antlio](https://github.com/antlio), it seems the URL can't be editted or manupulate anymore now. I guess that's the way move forward?

yes it is now read only from a reference, however if you encounter any issue within the client (web|desktop) please let me know!

**Comment by standleypg** [comment]
All good, we can close the issue now. Thank you so much for allocating some time to look into this!
